### PR TITLE
3.1-release-notes

### PR DESCRIPTION
**Pull request by resmaa**
**3.1 Release Notes**

Links to be updated

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1064#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
@resmaa: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | 5815fccb1fb5bdf7e5a417f89ff0c97452202722 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1064/librarium_build/1606284395298164736) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: content/docs/00-release-notes.md
Comment (karl-cardenas-coding) [inline content/docs/00-release-notes.md:34]:
```suggestion
* Palette App Mode now supports the use of [containers](). You can specify containers when creating an app profile.
```
I had single image, but we can actually add several containers

File: content/docs/00-release-notes.md
Comment (karl-cardenas-coding) [inline content/docs/00-release-notes.md:35]: What is the store in this context?

File: content/docs/00-release-notes.md
Comment (karl-cardenas-coding) [inline content/docs/00-release-notes.md:43]:
```suggestion
* Palette Tencent Clusters now support using [security groups]() for network isolation and improved security.
```

File: content/docs/00-release-notes.md
Comment (karl-cardenas-coding) [inline content/docs/00-release-notes.md:45]: What is app profile modeling? Is there a page we can link to? I'm not even sure I know what it is.

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]

File: content/docs/00-release-notes.md
Comment (resmaa) [reply content/docs/00-release-notes.md:35]: * Palette leverages the [Helm and OCI registries]() to configure, store and service packs.

**Review by karl-cardenas-coding — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1064#pullrequestreview-1229682869" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1064#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-1/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
